modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -345,6 +380,23 @@ private void _testSendNotification(
 			String to)
 		throws Exception {
 
+		_testSendNotification(
+			NotificationTemplateConstants.EDITOR_TYPE_FREEMARKER,
+			expectedNotificationQueueEntriesCount, expectedToEmailAddresses,
+			singleRecipient, to);
+
+		_testSendNotification(
+			NotificationTemplateConstants.EDITOR_TYPE_RICH_TEXT,
+			expectedNotificationQueueEntriesCount, expectedToEmailAddresses,
+			singleRecipient, to);
```

Comment:
Hey @seltong please move all tests related to freeMarker to a different test `testFreeMarkerNotification`, this approach I've used to create those tests are not good because one test is cover the entire sendNotification method and it should be splited so  we can divide the responsibilities.

Reply:
Done :+1: